enhancements/network/primary-user-defined-networks-for-virtualization-workloads.md

```diff
@@ -0,0 +1,638 @@
+
+N/A
+
+## Test Plan
+
+- E2E upstream and downstream jobs covering supported features across multiple
+networks.
+- E2E tests ensuring VM live-migration preserving established TCP connections. 
+- E2E tests covering reachability to external networks (VM egress over UDN
+networks for Layer2 topologies). 
+- E2E tests covering a VM exposed as a `LoadBalancer` service is reachable from
+the outside world. 
+- Scale testing to determine limits and impact of multiple user-defined
+networks.
+
+## Graduation Criteria
+
+### Dev Preview -> Tech Preview
+
+There will be no dev or tech preview for this feature.
+
+### Tech Preview -> GA
+
+Targeting GA in OCP version `4.17.Z`.
+
+### Removing a deprecated feature
+
+N/A
+
+## Upgrade / Downgrade Strategy
+
+This section only matters if we choose to release a version with bridge binding
+on 4.18, and later on migrate to passt.
+
+TODO: fill this section
+
+## Version Skew Strategy
+
+N/A
+
+## Operational Aspects of API Extensions
+
+Describe the impact of API extensions (mentioned in the proposal section, i.e. CRDs,
+admission and conversion webhooks, aggregated API servers, finalizers) here in detail,
+especially how they impact the OCP system architecture and operational aspects.
+
+- For conversion/admission webhooks and aggregated apiservers: what are the SLIs (Service Level
+  Indicators) an administrator or support can use to determine the health of the API extensions
+
+  Examples (metrics, alerts, operator conditions)
+  - authentication-operator condition `APIServerDegraded=False`
+  - authentication-operator condition `APIServerAvailable=True`
+  - openshift-authentication/oauth-apiserver deployment and pods health
+
+- What impact do these API extensions have on existing SLIs (e.g. scalability, API throughput,
+  API availability)
+
+  Examples:
+  - Adds 1s to every pod update in the system, slowing down pod scheduling by 5s on average.
+  - Fails creation of ConfigMap in the system when the webhook is not available.
+  - Adds a dependency on the SDN service network for all resources, risking API availability in case
+    of SDN issues.
+  - Expected use-cases require less than 1000 instances of the CRD, not impacting
+    general API throughput.
+
+- How is the impact on existing SLIs to be measured and when (e.g. every release by QE, or
+  automatically in CI) and by whom (e.g. perf team; name the responsible person and let them review
+  this enhancement)
+
+- Describe the possible failure modes of the API extensions.
+- Describe how a failure or behaviour of the extension will impact the overall cluster health
+  (e.g. which kube-controller-manager functionality will stop working), especially regarding
+  stability, availability, performance and security.
+- Describe which OCP teams are likely to be called upon in case of escalation with one of the failure modes
+  and add them as reviewers to this enhancement.
+
+## Support Procedures
+
+N/A
+
+## Alternatives
+
+### Binding mechanism
```

Comment:
This section will need to sync with the U/S Kubevirt proposal.
Hopefully that proposal will be posted to the Kubevirt community in a few days with the plan that fits the technical aspects, maintainability and the timeline limitations.